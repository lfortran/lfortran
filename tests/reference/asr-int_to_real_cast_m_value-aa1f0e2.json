{
    "basename": "asr-int_to_real_cast_m_value-aa1f0e2",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/int_to_real_cast_m_value.f90",
    "infile_hash": "0160ef255b4a2d056d9829227bbe241ee980f788e4ce7600d7f6e554",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-int_to_real_cast_m_value-aa1f0e2.stdout",
    "stdout_hash": "9657fb8a606d1895e25fa73269b9abcb18b209e916cf02815180fb32",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}