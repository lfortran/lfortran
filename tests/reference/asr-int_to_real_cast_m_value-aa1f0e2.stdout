(TranslationUnit
    (SymbolTable
        1
        {
            int_to_real_cast_m_value:
                (Program
                    (SymbolTable
                        2
                        {
                            a:
                                (Variable
                                    2
                                    a
                                    []
                                    Local
                                    (Cast
                                        (ArrayConstant
                                            12
                                            [1, 2, 3]
                                            (Array
                                                (Integer 4)
                                                [((IntegerConstant 1 (Integer 4) Decimal)
                                                (IntegerConstant 3 (Integer 4) Decimal))]
                                                FixedSizeArray
                                            )
                                            ColMajor
                                        )
                                        IntegerToReal
                                        (Array
                                            (Real 8)
                                            [((IntegerConstant 1 (Integer 4) Decimal)
                                            (IntegerConstant 3 (Integer 4) Decimal))]
                                            FixedSizeArray
                                        )
                                        (ArrayConstant
                                            24
                                            [1.0000000000000000e+00, 2.0000000000000000e+00, 3.0000000000000000e+00]
                                            (Array
                                                (Real 8)
                                                [((IntegerConstant 1 (Integer 4) Decimal)
                                                (IntegerConstant 3 (Integer 4) Decimal))]
                                                FixedSizeArray
                                            )
                                            ColMajor
                                        )
                                    )
                                    (ArrayConstant
                                        24
                                        [1.0000000000000000e+00, 2.0000000000000000e+00, 3.0000000000000000e+00]
                                        (Array
                                            (Real 8)
                                            [((IntegerConstant 1 (Integer 4) Decimal)
                                            (IntegerConstant 3 (Integer 4) Decimal))]
                                            FixedSizeArray
                                        )
                                        ColMajor
                                    )
                                    Save
                                    (Array
                                        (Real 8)
                                        [((IntegerConstant 1 (Integer 4) Decimal)
                                        (IntegerConstant 3 (Integer 4) Decimal))]
                                        FixedSizeArray
                                    )
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    int_to_real_cast_m_value
                    []
                    []
                )
        })
    []
)
